pkgs/tools/misc/rmlint/default.nix

```diff
@@ -1,26 +1,30 @@
-{ stdenv, fetchFromGitHub,
-  gettext, glib, json-glib, libelf, pkgconfig, scons, sphinx, utillinux }:
+{ stdenv, fetchFromGitHub
+, gettext, pkgconfig, scons
+, glib, json-glib, libelf, sphinx, utillinux }:
 
 with stdenv.lib;
 stdenv.mkDerivation rec {
   name = "rmlint-${version}";
-  version = "2.6.1";
+  version = "2.8.0";
 
   src = fetchFromGitHub {
     owner = "sahib";
     repo = "rmlint";
     rev = "v${version}";
-    sha256 = "1j09qk3zypw4my713q9g36kq37ggqd5v9vrs3h821p6p3qmmkdn8";
+    sha256 = "1gc7gbnh0qg1kl151cv1ld87vhpm1v3pnkn7prhscdcc21jrg8nz";
   };
 
-  configurePhase = "scons config";
+  CFLAGS="-I${stdenv.lib.getDev utillinux}/include";
```

Comment:
Does scons suspress our $NIX_CFLAGS_COMPILE variable? because adding `utillinux` to buildInputs will have the very same effect?

Reply:
Mhm. I added this patch myself :). So this probably not the case.